package.json

```diff
@@ -110,10 +110,12 @@
     "cucumber": "git+https://github.com/SebastienGllmt/cucumber-js.git",
     "eslint": "7.10.0",
     "eslint-config-airbnb": "18.2.0",
+    "eslint-config-prettier": "^6.15.0",
```

Comment:
Historically we prefer to use strict versions (remove the `^`) and manually validate any version upgrade.

Reply:
Sure